src/helpers/getDataType.js

```diff
@@ -35,7 +35,12 @@ function loadData(ELMFiles) {
       return value.libraryName && value.type === 'ExpressionRef' ? value : null;
     });
     expressionRefs.forEach((ref) => {
-      const libraryIdentifier = dependencyMap[ref.libraryName];
+      let libraryIdentifier;
+      if (dependencyMap[ref.libraryName].includes('connectathon')) {
+        libraryIdentifier = dependencyMap[ref.libraryName].substring(41);
+      } else {
+        libraryIdentifier = dependencyMap[ref.libraryName];
+      }
```

Comment:
I feel like we should brainstorm a more robust workaround here... I haven't thought of one off the top of my head yet. But we can talk about it